forms-flow-admin/src/components/users/users.tsx

```diff
@@ -372,22 +385,45 @@ const Users = React.memo((props: any) => {
       roles: selectedRolesIds,
       user: formData.user,
     };
-  CreateUser(
-    payload,
-    (data) => {
-      toast.success(t("User Added successfully!"));
-      clearForm();
-      closeInviteModal(); 
-      props.setInvalidated(true);
-    },
-    (err) => {
-      toast.error(t("User not exist!"));
-    }
-  );
+    CreateUser(
+      payload,
+      (data) => {
+        openSuccessModal();
+      },
+      (err) => {
+        setValidationError(t('User not exist!'));
+      }
+    );
   };
+  
 
   return (
     <>
+      <Modal
+        show={showSuccessModal}
+        onHide={closeSuccessModal}
+        centered
+        className="overflow-hidden">
+        <Modal.Header closeButton>
```

Comment:
Please make sure the success modal is closed on clicking outside of the modal

Reply:
Yes, its working so